crates/marine-test-macro-impl/src/errors.rs

```diff
@@ -50,6 +50,9 @@ pub enum TestGeneratorError {
 
     #[error("{0} is invalid UTF8 path")]
     InvalidUTF8Path(PathBuf),
+
+    #[error("a \"self\" argument found and it is not supported in test function")]
```

Comment:
You could use Rust [raw string literals](https://doc.rust-lang.org/reference/tokens.html#raw-string-literals) here in the following way:
`#[error(r#"a "self" argument found and it is not supported in test function"#)]`